Source/Core/DolphinQt/Config/Mapping/MappingIndicator.cpp

```diff
@@ -210,6 +244,24 @@ bool IsPointOutsideCalibration(Common::DVec2 point, ControllerEmu::ReshapableInp
   return current_radius > input_radius * ALLOWED_ERROR;
 }
 
+void DrawVirtualNotches(QPainter& p, ControllerEmu::ReshapableInput& stick, QColor gate_brush_color)
+{
+  const double segment_size = stick.GetVirtualNotchSize();
+  if (segment_size <= 0.0)
+    return;
+
+  p.setBrush(gate_brush_color.darker(125));
```

Comment:
This looks bad in dark themes. You could `p.setBrush(gate_pen_color);` in `DrawReshapableInput` right before the `DrawVirtualNotches` call instead.